### PR TITLE
llgo 0.8.8 (new formula)

### PR DESCRIPTION
**Pull request by cpunion**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/l/llgo.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
+class Llgo < Formula
+  desc "Go compiler based on LLVM integrate with the C ecosystem and Python"
+  homepage "https://github.com/goplus/llgo"
+  url "https://github.com/goplus/llgo/archive/refs/tags/v0.8.8.tar.gz"
+  sha256 "be6979d47bcc8f89f2156f6b2d6f603854104fea7452519bac144400fbecabf0"
+  license "Apache-2.0"
+
+  depends_on "go"
+  depends_on "llvm@17"
```
Comment (chenrui333) [inline Formula/l/llgo.rb:9]: any issue tracking llvm18 support?

Comment (cpunion) [reply Formula/l/llgo.rb:9]: Current version only works with llvm@17, will fix in next release.

**Comment by cpunion** [comment]
@chenrui333 Still has a style problem, but I can't find it. No errors when I run `brew audit` locally.

```shell
brew audit --strict --online llgo
brew audit --new --online llgo
brew audit --new --formula llgo
```

What else am I missing?

**Comment by carlocab** [comment]
Squash all your commits together and use the commit message
```
llgo 0.8.8 (new formula)
```

File: Formula/l/llgo.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/l/llgo.rb:11]: when would be the next release? maybe better wait for that one?

Comment (cpunion) [reply Formula/l/llgo.rb:11]: OK, will update after next major release. But maybe need lock to `llvm@18`.

Comment (chenrui333) [reply Formula/l/llgo.rb:11]: I actually dont think we are able to pin the llvm version to 18 (maybe good left a note to update to it once llvm 19 is out)

now we are talking about this, I feel like we can just ship with llvm 17

**Review by chenrui333 — APPROVED** [review]
lgtm

**Comment by chenrui333** [comment]
@cpunion, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Comment by cpunion** [comment]
@chenrui333 Thanks for approval.

**Comment by cpunion** [comment]
@chenrui333 I updated my local `homebrew-core` and I can see the `llgo.rb` file is updated with bottle section, but can't search and install it, are there some cache need to clean?

```shell
(base) ➜  ~ brew search llgo
==> Formulae
lego

==> Casks
font-ofl-sorts-mill-goudy-tt                                                                                          font-sorts-mill-goudy
(base) ➜  ~ brew install llgo
Warning: No available formula with the name "llgo". Did you mean lego?
==> Searching for similarly named formulae and casks...
==> Formulae
lego

To install lego, run:
  brew install lego

==> Casks
font-ofl-sorts-mill-goudy-tt                                                                                          font-sorts-mill-goudy

To install font-ofl-sorts-mill-goudy-tt, run:
  brew install --cask font-ofl-sorts-mill-goudy-tt
(base) ➜  ~
```

**Comment by cpunion** [comment]
Resolved, seems it's cache of Homebrew server.

**Pull request merged**